python_modules/dagster/dagster_tests/core_tests/storage_tests/test_fs_io_manager.py

```diff
@@ -100,3 +102,35 @@ def get_solid_version(self, _):
             result = my_job.execute_in_process(instance=instance)
             assert result.success
             assert len(recorder) == 1
+
+
+def test_fs_io_manager_unpicklable():
+    @op
+    def unpicklable_output():
+        # lambda functions can't be pickled
+        return lambda x: x*x
```

Comment:
probably worth testing the full gamut of pickle errors that we can get, ie recursion error, direct unpickleable error, etc

Reply:
yup - added them in